Gemfile.lock

```diff
@@ -265,14 +288,14 @@ DEPENDENCIES
   ims-lti (= 1.1.13)
   jquery-rails (~> 4.2.2)
   jquery-ui-rails (~> 4.2.0)
-  json (= 2.0.3)
+  json
```

Comment:
Maybe ~> 2.6 to avoid unintentional upgrade to 3.0 which by definition is not backward compatible

Reply:
The change is of course to the gemfile, not this file